textile/features.textile

```diff
@@ -2276,6 +2436,60 @@ class PresenceMessage // TP*
   timestamp: Time // TP3g
   memberKey() -> String // TP3h
 
+class ObjectMessage // OM*, internal
+  id: String // OM2a
+  clientId: String // OM2b
+  connectionId: String // OM2c
+  extras: JsonObject? // OM2d
+  timestamp: Time // OM2e
+  operation: ObjectOperation? // OM2f
+  object: ObjectState? // OM2g
+  serial: String // OM2h
+  siteCode: String // OM2i
+
+class ObjectOperation // OOP*, internal
+  action: ObjectOperationAction // OOP3a
+  objectId: String // OOP3b
+  mapOp: MapOp? // OOP3c
+  counterOp: CounterOp? // OOP3d
+  map: Map? // OOP3e
+  counter: Counter? // OOP3f
+  nonce: String? // OOP3g
+  initialValue: Binary? // OOP3h
+  initialValueEncoding: String? // OOP3i
+
+class ObjectState // OST*, internal
+  objectId: String // OST2a
+  siteTimeserials: Dict<String, String> // OST2b
+  tombstone: Boolean // OST2c
+  createOp: ObjectOperation? // OST2d
+  map: Map? // OST2e
+  counter: Counter? // OST2f
+
+class MapOp // MOP*, internal
+  key: String // MOP2a
+  data: ObjectData? // MOP2b
+
+class CounterOp // COP*, internal
+  amount: Number // COP2a
+
+class Map // MAP*, internal
```

Comment:
This is called `ObjectMap` in ably-js

Reply:
I would prefer `ObjectMap` and `ObjectCounter` too

Reply:
I'm happy with those proposed names, yep 👍